lib/routes.js

```diff
@@ -35,8 +35,8 @@ const statsClient = new StatsClient(localCacheClient, STATS_INTERVAL,
     STATS_EXPIRY);
 
 function checkUnsuportedRoutes(req, res, log) {
-    if (req.query.policy !== undefined ||
-        req.query.tagging !== undefined) {
+    if (utils.isUnsupportedQuery(req.queryStr)) {
```

Comment:
will you handle the filter for the post routes that do not contain our two supported queries in a separate pr?  That's fine just want to make sure it gets picked up.